README.md

```diff
@@ -211,8 +211,35 @@ Usage: sparql_runner [OPTIONS] [MAPPING_SUITE_ID]
 Options:
   -m, --opt-mappings-folder TEXT
   --help                                      Show this message and exit.
+```
 
+### API
+#### Start local API server
+To start the API server:
+```bash
+api-start-server
```

Comment:
very well, however, will need to become a service started with Docker. We need to involve Dragos in deploying this. To get inspired on how docker files are written, you can look into any MDR project (rdf-validator-ws, rdf-fingerprinter-ws, rdf-differ-ws)